src/site/stages/build/process-cms-exports/schemas/transformed/node-event_listing.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-event_listing'] },
+    entityBundle: { enum: ['event_listing'] },
+    entityType: { enum: ['node'] },
+    entityUrl: {
+      // Probably should pull this out into a common schema
```

Comment:
@rianfowler I accidentally deleted your comment here when I meant to delete my own (trying to adjust the order of things).  Here is what I said:

We have an [`EntityUrl`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/process-cms-exports/schemas/common/entity-url.js) common schema.  It doesn't require some of the internal breadcrumb properties at the moment, but that will change once we actually get that piece from the CMS.

Reply:
thanks added